sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -630,6 +633,53 @@ public static void printHelp(PrintStream out, Class<? extends PipelineOptions> i
     }
   }
 
+  public static Map<String, PipelineOptionDescriptor> describe(Set<Class<? extends PipelineOptions>> ifaces) {
```

Comment:
This should probably go into `PipelineOptionsTranslation`.

Reply:
It's not really about translating pipeline options, but providing meta data about it. PipelineOptionsFactory already has all the utility code for it.